gradle/libs.versions.toml

```diff
@@ -93,6 +94,7 @@ square-sqldelight-native-driver = { module = "com.squareup.sqldelight:native-dri
 square-sqldelight-plugin = { module = "com.squareup.sqldelight:gradle-plugin", version.ref = "sqldelight" }
 square-sqldelight-runtime = { module = "com.squareup.sqldelight:runtime", version.ref = "sqldelight" }
 square-sqldelight-sqlite-driver = { module = "com.squareup.sqldelight:sqlite-driver", version.ref = "sqldelight" }
+tournament-bracket = { module = "com.adammcneilly.tournamentbracket:tournamentbracket", version.ref = "tournamentBracket" }
```

Comment:
This is only published to local maven, which means GH Actions will fail until we move this to a public maven repo.